docs/DeveloperGuide.md

```diff
@@ -284,9 +287,9 @@ The implementation of `Model#unlinkFriend()` is as follows:
 
 ![Implementation of unlink command in model](images/UnlinkSequenceDiagram.png)
 
-### Schedule Feature
+### 4.4 Schedule Feature
 
-#### Implementation
+#### 4.4.1 Implementation
```

Comment:
Remember to update the link here as well as in other places where other parts of the DG are referenced..

>  Schedule has the command format of `friend --schedule FRIEND_ID -p START_HOUR END_HOUR DAY -f IS_FREE`, and as in  **[Logic component] (#logic-component)**,